### ISSUE TITLE
Text inside definition list does not render correctly

### ISSUE THREAD

**Issue by foo-dogsquared** [rasendubi/uniorg#15]
[Reference](https://orgmode.org/manual/Plain-Lists.html#Plain-Lists)

>  Description list items are unordered list items, and contain the separator ‘::’ to distinguish the description term from the description.

I've used the `example` from the examples folder as the setup.
The description is rendering but the description term does not.

Example input:

```org
- [[https://example.com][Example]] :: Hello there!
- [[https://github.com][GitHub]] :: This is GitHub, your hub for Git repos.
- *Gitlab* :: Alternative to GitHub
- /Sourcehut/ :: Another alternative to GitHub that primarily uses email-based workflows.
- /Codeberg/ :: *ANOTHER ALTERNATIVE*
- /*self-hosting Git server*/ :: /*The ultimate Git solution for privacy-oriented individuals!*/
```

Output (formatted to make it easier to read):

```html
<dl>
  <dt>[[https://example.com][Example]]</dt><dd><p>Hello there!</p></dd>
  <dt>[[https://github.com][GitHub]]</dt><dd><p>This is GitHub, your hub for Git repos.</p></dd>
  <dt>*Gitlab*</dt><dd><p>Alternative to GitHub</p></dd>
  <dt>/Sourcehut/</dt><dd><p>Another alternative to GitHub that primarily uses email-based workflows.</p></dd>
  <dt>/Codeberg/</dt><dd><p><strong>ANOTHER ALTERNATIVE</strong></p></dd>
  <dt>/*self-hosting Git server*/</dt><dd><p><em><strong>The ultimate Git solution for privacy-oriented individuals!</strong></em></p></dd>
</dl>
```

The expected output (extracted from the HTML exporter from Emacs):

```html
<dl>
  <dt><a href="https://example.com">Example</a></dt><dd>Hello there!</dd>
  <dt><a href="https://github.com">GitHub</a></dt><dd>This is GitHub, your hub for Git repos.</dd>
  <dt><b>Gitlab</b></dt><dd>Alternative to GitHub</dd>
  <dt><i>Sourcehut</i></dt><dd>Another alternative to GitHub that primarily uses email-based workflows.</dd>
  <dt><i>Codeberg</i></dt><dd><b>ANOTHER ALTERNATIVE</b></dd>
  <dt><i><b>self-hosting Git server</b></i></dt><dd><i><b>The ultimate Git solution for privacy-oriented individuals!</b></i></dd>
</dl>
```

It works with ordinary unordered lists so I figured this would be a bug.

**Comment by rasendubi** [comment]
Yeah, I noticed this before, however it matches how org-element.el parses
the same input (it does not parse definition terms further), so I didn't
fix it.

Though I didn't know that export to html parses terms further. Thanks for
checking this!

I'll fix this issue when I'm back from vacation (next Monday)

**Comment by rasendubi** [comment]
Fixed in #16 and published as v0.3.0.
